get_data/graphs_for_server.py

```diff
@@ -0,0 +1,85 @@
+import numpy as np
+import pandas as pd
+import matplotlib
+from matplotlib import pyplot as plt
+
+
+def plot_to_file(filename, predictions, sell_today, todays_price):
+    """
+    Reads a pandas dataframe with all dates as datetime objects(!), and returns an .SVG
+    that contains the predicted values for the next few days, along with the profit
+    one can expect if one waited until that day (as costs go up daily).
+    filename does not need to contain the .svg file extension.
+    """
+    # Read in files
+    SERVER_URL = "http://35.204.193.240/"
+    n_predictions = len(predictions.index)
```

Comment:
is an extra argument in the function more efficient than having just an extra line somewhere? like is there a hierarchy of where it is more acceptable to replete yourself ?

Reply:
It's just good practice to use abstracted stuff where you have implemented it, otherwise you unnecessarily repeat yourself. It also means, if for example somebody changes that url in the main script assuming it has been used dynamically all over, the changes will trickle down - it saves having to go through every function and changing it :)